packages/cli/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "CLI for building for and deploying to The Graph",
   "license": "(Apache-2.0 OR MIT)",
   "engines": {
-    "node": ">=14"
+    "node": ">=18"
```

Comment:
This would be considered a breaking change.

Reply:
Agree with @enisdenjo 
Should release major in this case

Reply:
we are still on `0.x` so it is a breaking change in that sense :)